src/render/render_master.c

```diff
@@ -4,15 +4,48 @@
 #include <stdlib.h>
```

Comment:
Can the external includes be in alphabetical order pls?

Reply:
This should hopefully be done now